### ISSUE TITLE
WandbCallback cannot log multiple metrics

### ISSUE THREAD

**Issue by rsomani95** [fastai/fastai#3024]
Please confirm you have the latest versions of fastai, fastcore, fastscript, and nbdev prior to reporting a bug (delete one): YES

**Describe the bug**
`fastai.callback.wandb.WandbCallback` throws a `TypeError` when trying to log multiple metrics. This happens in the `[_format_config_value[item] for item in v]` line [here](https://github.com/fastai/fastai/blob/c1974d89acbe452ced52ca5c37f8918e49345f81/fastai/callback/wandb.py#L169)

**To Reproduce**
```python
from fastai.vision.all import *
from fastai.callback.wandb import _format_config_value, _format_config

config_pass = {'metrics': accuracy}
_format_config(config_pass) # no problem

config_fail = {'metrics': [accuracy]}
_format_config(config_fail) # throws `TypeError`
```

**Expected behavior**
The list of metrics is logged without any issues

**Error with full stack trace**

```python
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-152-28229ec88a9e> in <module>
----> 1 _format_config_value([accuracy, accuracy])

~/miniconda3/envs/det/lib/python3.7/site-packages/fastai/callback/wandb.py in _format_config_value(v)
    167 def _format_config_value(v):
    168     if isinstance(v, list):
--> 169         return [_format_config_value[item] for item in v]
    170     elif hasattr(v, '__stored_args__'):
    171         return {**_format_config(v.__stored_args__), '_name': v}

~/miniconda3/envs/det/lib/python3.7/site-packages/fastai/callback/wandb.py in <listcomp>(.0)
    167 def _format_config_value(v):
    168     if isinstance(v, list):
--> 169         return [_format_config_value[item] for item in v]
    170     elif hasattr(v, '__stored_args__'):
    171         return {**_format_config(v.__stored_args__), '_name': v}

TypeError: 'function' object is not subscriptable
```

**Comment by rsomani95** [comment]
@borisdayma

**Comment by rsomani95** [comment]
I think the fix is:
```python
def _format_config_value(v):
    if isinstance(v, list):
        #return [_format_config_value[item] for item in v]
        return [_format_config_value(item) for item in v]
    elif hasattr(v, '__stored_args__'):
        return {**_format_config(v.__stored_args__), '_name': v}
    return v
```

**Comment by rsomani95** [comment]
Sorry, this was a silly oversight on my part. I was running 3.1.5 and just realised this was fixed in 3.1.6

**Comment by borisdayma** [comment]
No problem, feel free to reach back if you have any issue
